db/migrate/20230420233246_create_blobs.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBlobs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :blobs do |t|
+      t.string :document_id
+      t.string :document_filename
+      t.text :data
```

Comment:
As mentioned, I like having this stored in the DB, but I think you'd be better off storing this as a JSON datatype instead of text.  That would cut down on some future manipulation that you'd need in order to query the JSON later by casting the text as json.  But it works.

Reply:
I agree so, I added a migration to change the datatype of data column